src/package-install-scripts.js

```diff
@@ -34,6 +34,10 @@ export default class PackageInstallScripts {
   force: boolean;
   artifacts: InstallArtifacts;
 
+  setForce(force: boolean) {
```

Comment:
Good question.

The force property was formerly only set on construction of the `PackageInstallScripts` class:
https://github.com/yarnpkg/yarn/blob/master/src/cli/commands/install.js#L178

When the install bailout method runs the `PackageInstallScripts` instance has already been created. Adding the setter, allowing overrides, seemed like a simpler solution rather than reinstantiating. Happy to hear another approach.